source/utilities.tex

```diff
@@ -6140,6 +6146,44 @@
 no complexity requirements.
 \end{itemdescr}
 
+\indexlibrarymember{visit}{variant}%
+\begin{itemdecl}
+template<class Self, class Visitor>
+  constexpr decltype(auto) visit(this Self&& self, Visitor&& vis);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+Let \tcode{V} be
+\tcode{\exposid{OVERRIDE_REF}(Self\&\&, \exposid{COPY_CONST}(remove_reference_t<Self>, variant))}\iref{forward}.
+
+\pnum
+\constraints
+The call to \tcode{visit} does not use
+an explicit \grammarterm{template-argument-list} that
+begins with a type \grammarterm{template-argument}.
+
+\pnum
+\effects
+Equivalent to: \tcode{return std::visit(std::forward<Visitor>(vis), (V)self);}
```

Comment:
Is the use of `std::visit` here deliberate/important? Usually we don't qualify names within `std`.

Reply:
Without qualification, `visit` here would refer to the member function template `std::variant::visit` instead of the namespace-scope template `std::visit`, making the equivalent code ill-formed.

Reply:
Ah of course, thanks!